mux.go

```diff
@@ -47,7 +47,7 @@ type Mux struct {
 // NewMux returns a newly initialized Mux object that implements the Router
 // interface.
 func NewMux() *Mux {
-	mux := &Mux{tree: &node{}}
+	mux := &Mux{tree: &node{}, pool: new(sync.Pool)}
```

Comment:
minor style thing, can you change this to: `pool: &sync.Pool{}`

Reply:
Sure thing.

Reply:
Done.